script/verify.py

```diff
@@ -10,6 +10,8 @@
 
 from pymongo import MongoClient
 
+from oauth.google_auth_service import GoogleAuthService, UserNotValidatedException
```

Comment:
typo in import, OAuth should be a package (add init file in that)

Reply:
done